public/rulesets/default/2_3_3_Request_body

```diff
@@ -0,0 +1,41 @@
+rules: 
+  - id: "2.3.3"
+    title: "Support JSON/BSON Encapsulation in Request Body"
+    message: "The request body must support both JSON and BSON encapsulation. JSON must conform to key:value format, and BSON-specific types must be correctly declared when 'application/bson' is used."
```

Comment:
BSON type won't be supported in first version as we don't have such API examples.
for the JSON type, please add the requirements that under content there must be "application/json" key:

```
      requestBody:
        description: Association object that needs to be created
        content:
          application/json:
```